### PR TITLE
fix: trying should(be.visible) on annotations flakey test

### LINKED ISSUE #1370: Flaky Test: can create an annotation that is scoped to a dashboard cell
the test **can create an annotation that is scoped to a dashboard cell** is flaky.

Source: https://github.com/influxdata/ui/blob/master/cypress/e2e/cloud/annotations.test.ts#L252

Example:
- https://app.circleci.com/pipelines/github/influxdata/monitor-ci/773/workflows/8c06d1f7-4034-46ea-9af0-59b4afd5824b/jobs/6692
- https://app.circleci.com/pipelines/github/influxdata/monitor-ci/769/workflows/1cd146bd-53a0-4e0c-8c90-786d3e6ae345/jobs/6663
- https://app.circleci.com/pipelines/github/influxdata/monitor-ci/998/workflows/3c8c1405-c7eb-4f62-bfab-b9f25413adca/jobs/8690
- https://app.circleci.com/pipelines/github/influxdata/monitor-ci/1204/workflows/28a750a5-8f34-4c7a-a996-b42096ba2e23/jobs/10243/artifacts

image

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #1370

Round 2 of trying to fix flakey annotations test. Following other tests that use .should(be.visible)

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 54 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1716)

File: cypress/e2e/cloud/annotations.test.ts [hunk 1]
```diff
@@ -543,12 +543,18 @@ describe('The Annotations UI functionality', () => {
       cy.getByTestID('button')
         .click()
         .then(() => {
-          cy.getByTestID('selector-list schmucket').click()
-          cy.getByTestID(`selector-list m`).click()
+          // Look at this article for removing flakiness https://www.cypress.io/blog/2020/07/22/do-not-get-too-detached/
+          cy.getByTestID('selector-list schmucket')
+            .should('be.visible')
+            .click()
```
Comment (genehynson) [inline cypress/e2e/cloud/annotations.test.ts:549]: I like the idea of adding `.should('be.visible')` but I think we should avoid chaining it. I'd recommend:
```
cy.getByTestID('selector-list schmucket').should('be.visible')
cy.getByTestID('selector-list schmucket').click()
```

Comment (zoesteinkamp) [reply cypress/e2e/cloud/annotations.test.ts:549]: sure, im down with some non chaining, let me make the switch

**Review by genehynson — COMMENTED** [review]
Looks good! Can we fix the chaining in this section too?

```
      cy.getByTestID('overlay--container').within(() => {
        cy.getByTestID('edit-annotation-message')
          .should('be.visible')
          .click()
          .focused()
          .type('annotation in newCell')
        cy.getByTestID('annotation-submit-button').click()
      })
```

**Review by genehynson — APPROVED** [review]
Thanks! Hopefully this helps! 🤞

**Pull request merged**